README.md

```diff
@@ -52,42 +52,67 @@ $ sql-language-server up --method stdio
-    "dbHost": "127.0.0.1",
-    "port": 3306,
-    "identityFile": "~/.ssh/id_rsa",
-    "passphrase": "123456"
-  }
+  "connections": [
+    {
+      "name": "sql-language-server",
+      "adapter": "mysql",
+      "host": "localhost",
+      "port": 3307,
+      "user": "username",
+      "password": "password",
+      "database": "mysql-development",
+      "projectPaths": ["/Users/joe-re/src/sql-language-server"],
+      "ssh": {
+        "user": "ubuntu",
+        "remoteHost": "ec2-xxx-xxx-xxx-xxx.ap-southeast-1.compute.amazonaws.com",
+        "dbHost": "127.0.0.1",
+        "port": 3306,
+        "identityFile": "~/.ssh/id_rsa",
+        "passphrase": "123456"
+      }
+    },
+    {
+      "name": "postgres-project",
+      "adapter": "prostgres",
+      "host": "localhost",
+      "port": 5432,
+      "user": "postgres",
+      "password": "pg_pass",
+      "database": "pg_test",
+      "projectPaths": ["/Users/joe-re/src/postgres_ptoject"]
+    }
+  ]
 }
 ```
 
 Please restart sql-language-server process after create .sqlrc.json.
 
 #### Parameters
 
-| Key      | Description                 | value                   | required | default                           |
-| -------- | --------------------------- | ----------------------- | -------- | --------------------------------- |
-| adapter  | Database type               | `"mysql" | "postgres"`  | true     |                                   |
-| host     | Database host               | string                  | true     |                                   |
-| port     | Database port               | string                  | false    | mysql:3306, postgres:5432         |
-| user     | Database user               | string                  | true     | mysql:"root", postgres:"postgres" |
-| password | Database password           | string                  | false    |                                   |
-| database | Database name               | string                  | false    |                                   |
-| ssh      | Settings for port fowarding | \*see below SSH section | false    |                                   |
+| Key          | Description                                                                                                               | value                   | required | default                           |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------- | ----------------------- | -------- | --------------------------------- |
+| name         | Connection name(free-form text)                                                                                           |                         | true     |                                   |
+| adapter      | Database type                                                                                                             | `"mysql" | "postgres"`  | true     |                                   |
```

Comment:
nit: We can not use 'vertical bar' in the Markdown table. But we can use character reference.
```suggestion
| adapter      | Database type                                                                                                             | "mysql" #124; "postgres"  | true     |                                   |
```

Reply:
ah I see, thanks 🙏